cmd/provisioner-localpv/app/helper_blockdevice.go

```diff
@@ -63,6 +63,8 @@ type HelperBlockDeviceOptions struct {
 	capacity     string
 	//	deviceType string
 	bdcName string
+	//  volumeMode of PVC
+	volumeMode corev1.PersistentVolumeMode
```

Comment:
createBlockDeviceClaim - should request for block devices that are not mounted - if the volumeMode is block.  A flag may have to be set on the BDC or verified once a BD is returned. cc: @akhilerm @mittachaitu did something similar for cstor pools.

Reply:
There is a field [`.spec.deviceClaimDetails.blockVolumeMode`](https://github.com/openebs/node-disk-manager/blob/56a2deb43eff0361e40d885e6f208341b6b34afd/pkg/apis/openebs/v1alpha1/blockdeviceclaim_types.go#L83) in BDC Spec. It can be set to `Block`, then NDM will give only those disks which does not have a mountpoint and a filesystem.